flavors/python-3.6-data-science-EXASOL-6.2.0/flavor_base/security_scan/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM {{release}}
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN mkdir -p /build_info/packages
+COPY security_scan/packages /build_info/packages/security_scan
+RUN /scripts/install_scripts/install_via_apt.pl --file /build_info/packages/security_scan/apt_get_packages_trivy_deps
+RUN wget -qO - https://aquasecurity.github.io/trivy-repo/deb/public.key | apt-key add - && \
```

Comment:
curl is I think everywhere installed, so maybe use instead curl

Reply:
I think this is a good idea

Reply:
Done